plugins/modules/packaging/os/rhn_channel.py

```diff
@@ -46,6 +46,12 @@
         description:
             - RHN/Satellite password.
         required: true
+    validate_certs:
+        description:
+            - If C(no), SSL certificates will not be validated.
+            - This should only set to C(no) when used on self controlled sites using self-signed certificates.
```

Comment:
```suggestion
            - This should only set to C(no) when used on self controlled sites using self-signed certificates.
```
I would make the warning even scarier. Maybe:
```suggestion
            - This should only set to C(no) when used on self controlled sites using self-signed certificates, and you are absolutely sure that nobody can modify traffic between the module and the site.
```